composer.json

```diff
@@ -117,7 +117,7 @@
             "Composer\\Config::disableProcessTimeout",
             "phpunit"
         ],
-        "verify-callmap": "phpunit tests/Internal/Codebase/InternalCallMapHandlerTest.php",
+        "verify-callmap": "@php phpunit tests/Internal/Codebase/InternalCallMapHandlerTest.php",
```

Comment:
This allows me to test multiple php versions. Thought I'd try to commit it instead of changing it every time.